### ISSUE TITLE
Use blockmaps or some form of diffing/compression to updates

### ISSUE THREAD

**Issue by Nokel81** [kontena/lens#5065]
**What would you like to be added**:
The ability to have updates be in the form of block maps, or partial updates along with compression formats.

**Why is this needed**:
To reduce the server resources when pushing updates.

**Comment by jakolehm** [comment]
Not sure if we need this once we have better CDN in place?

**Comment by Nokel81** [comment]
Fair enough, this could be considered lower priority then. Especially since I don't know if it is implemented for all platforms yet in `electron-builder`
